features/support/helpers/session_helpers.rb

```diff
@@ -14,7 +14,7 @@ def try_buyer_login_internal(username, password)
 
   def try_buyer_login_oauth
     # it works for Oauth2, which is for what is being used. In case it wants to be used to Auth0, it needs the state param
-    visit "/auth/#{@authentication_provider.system_name}/callback"
+    visit "/auth/#{@authentication_provider.system_name}/callback?code=foo"
```

Comment:
why wasn't this there before?.. or why did you need to add it?

Reply:
Before this change, parameters where not used to determine which auth strategy was tried, It was always `:provider_oaut2`.

Now, we identify an `Oauth2` strategy when the controller receives `system_name` and `code`. Lacking of `code` made the test fail.

It worked before, maybe the `code` param it's not really needed for `Oauth2`, or maybe the tests using this apply some mocks... I don't know.